### DISCUSSION TITLE
Would it be beneficial for us to go through the aks-secure-baseline as a team?

### DISCUSSION THREAD

**Discussion by AAkindele** [retaildevcrews/ngsa#571] [PnP Deploy]
@retaildevcrews/fc-pnp-deploy
Hey  team. Would it be beneficial for us to walk through the default [aks-secure-baseline](https://github.com/mspnp/aks-secure-baseline) setup as a team? I'm thinking one person can drive. The rest can take notes or focus on getting more context on the baseline setup. The goal is to help give the team more context around our starting point. Other suggestions are also welcome.

Side note: I'm experimenting with the GitHub discussions feature and thought this topic would be a good use case😄.

**Comment by atxryan** [comment]
Yes, and as always hit that record button!
